### PR TITLE
Added CI doc

### PR DESCRIPTION
**Pull request by slintes**

### CODE REVIEW

File: docs/ci.md [hunk 1]
```diff
@@ -0,0 +1,59 @@
...
+
+- several Dockerfile located in the [openshift-ci](https://github.com/openshift-kni/performance-addon-operators/tree/master/openshift-ci) directory:
+    - Dockerfile.tools: this image is used for building the binaries
+    - Dockerfile.deploy: this image is used for building the operator image
+    - Dockerfile.registry.*: these images are used for building the registry image:
+        - intermediate: this is used as base image
+        - build: this is used during CI run
+        - upstream.dev: this is used for being pushed to quay.io as snapshot images, see mirroring config below
+        
+> Note: Openshift CI can only use Dockerfiles from the git repository. You can't create Dockerfiles on the file during the CI run
```
Comment (gsr-shanks) [inline docs/ci.md:16]: Typo: on the fly

Comment (slintes) [reply docs/ci.md]: thx, fixed

**Pull request merged**